### PR TITLE
esp_lvgl_port: Backport changes for IDF v4.4

### PR DESCRIPTION
**Pull request by espzav**
Updated LVGL port component with latest backported changes in IDF 4.4.4

### CODE REVIEW

**Comment by espzav** [comment]
@tore-espressif PTAL

File: components/esp_lvgl_port/esp_lvgl_port.c [hunk 1]
```diff
@@ -23,12 +23,6 @@
 #include "esp_lcd_touch.h"
 #endif
 
-#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 0, 1)
```
Comment (igrr) [inline components/esp_lvgl_port/esp_lvgl_port.c:26]: You still need to check if IDF is 4.4.4 or earlier (e.g. v4.4.3), don't you?

File: components/esp_lvgl_port/esp_lvgl_port.c [hunk 2]
```diff
@@ -23,7 +23,7 @@
...
+#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(4, 4, 4)
```
Comment (tore-espressif) [inline components/esp_lvgl_port/esp_lvgl_port.c:26]: Unfortunately, the feature with esp-lcd `esp_lcd_panel_io_register_event_callbacks` is not included in v5.0.0. 

So it should work on:  <v4.4.4, v5.0.0) ∪ <v5.0.1, ∞)

Comment (tore-espressif) [reply components/esp_lvgl_port/esp_lvgl_port.c:26]: The CI did not catch it because we test on release _branches_ rather than on release _tags_

**Review by tore-espressif — CHANGES_REQUESTED** [review]
We must make sure that the code is compatible with esp-idf tag v5.0

File: components/esp_lvgl_port/esp_lvgl_port.c (hunk 1 above)
Comment (espzav) [reply components/esp_lvgl_port/esp_lvgl_port.c:26]: Yes, fixed.

File: components/esp_lvgl_port/esp_lvgl_port.c (hunk 2 above)
Comment (espzav) [reply components/esp_lvgl_port/esp_lvgl_port.c]: You are right. Fixed.

**Comment by espzav** [comment]
Tested with master, v5.0, v4.4.4, v4.4.3

**Review by tore-espressif — APPROVED** [review]
LGTM!

**Pull request merged**